cmd/dellhw_exporter/dellhw_exporter.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/galexrt/dellhw_exporter/collector"
 	"github.com/galexrt/dellhw_exporter/pkg/omreport"
+	"github.com/kardianos/service"
```

Comment:
I saw that this package also supports systemd under Linux.
Would there be an easy way to expose the `program` for Windows service and Linux as well?

Reply:
The kardianos/service module provides several functions that aids in running or creating services. This MR only uses the Run() method, which is used for both Linux and Windows. By using Run(), we enable the binary to respond to the non-trivial callback system Windows requires for it's services. When running under Linux, I believe the functionality remains unchanged.

Install(), Uninstall(), and several other function are available, but I imagine these would be utilized in some sort of installer binary. Is that what you are looking for here?